elastalert/alerters/line.py

```diff
@@ -14,7 +14,14 @@ def __init__(self, rule):
         self.linenotify_access_token = self.rule.get("linenotify_access_token", None)
 
     def alert(self, matches):
-        body = self.create_alert_body(matches)
+        body = ''
+        for match in matches:
+            body += str(BasicMatchString(self.rule, match))
+            if len(matches) > 1:
+                body += '\n----------------------------------------\n'
+        if len(body) > 999:
+            body = body[0:900] + '\n *message was cropped according to line notify embed description limits!* '
+        body += '```'
```

Comment:
What is the purpose of the 3 backticks at the end of the body, on this PR and the one for Chatwork?

Reply:
I erased. It's garbage. Discord also has similar trash. discord will be fixed tomorrow.